Sources/Lumina/Camera/Extensions/CapturePhotoExtension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  CapturePhotoExtension.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+@available (iOS 11.0, *)
+extension AVCapturePhoto {
+  func normalizedImage(forCameraPosition position: CameraPosition) -> UIImage? {
+    LuminaLogger.notice(message: "normalizing image from AVCapturePhoto instance")
+    guard let cgImage = self.cgImageRepresentation() else {
+      return nil
+    }
+    return UIImage(cgImage: cgImage, scale: 1.0, orientation: getImageOrientation(forCamera: position))
+  }
+  
+  private func getImageOrientation(forCamera: CameraPosition) -> UIImage.Orientation {
+    switch UIApplication.shared.statusBarOrientation {
+      case .landscapeLeft:
+        return forCamera == .back ? .down : .upMirrored
+      case .landscapeRight:
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/CapturePhotoExtension.swift#L26"></span>Case statements should vertically align with their enclosing switch statement.<br />
<code>switch_case_alignment</code> <code>CapturePhotoExtension.swift:26</code></td>
    </tr>
  </tbody>
</table>